### PR TITLE
ENG-3298_Smtp_Server_Tests

### PR DESCRIPTION
**Pull request by GianMartinoEntando**

### CODE REVIEW

File: ui-tests/cypress/support/pageObjects/administration/emailConfiguration/EmailConfigurationPage.js [hunk 1]
```diff
@@ -1,3 +1,98 @@
 import Content from '../../app/Content.js';
+import { htmlElements } from '../../WebElement';
+//import DeleteDialog from '../../app/DeleteDialog';
```
Comment (g-di-martino) [inline ui-tests/cypress/support/pageObjects/administration/emailConfiguration/EmailConfigurationPage.js:3]: Please, remove commented code

File: ui-tests/cypress/support/pageObjects/administration/emailConfiguration/EmailConfigurationPage.js [hunk 2]
```diff
@@ -1,3 +1,98 @@
...
+export default class EmailConfigurationPage extends Content {
+
+
+    emailConfigurationTabList = `${htmlElements.ul}[role=tablist]`;
+    formGroup= `${htmlElements.div}.form-group`;
+    toolButton = `${htmlElements.div}.btn-toolbar.pull-right`;
+    switch = `${htmlElements.div}.SwitchRenderer`;
+    switchContainer = `${htmlElements.div}[class="bootstrap-switch-container"]`;
+
+getSenderManagementTab(){
```
Comment (g-di-martino) [inline ui-tests/cypress/support/pageObjects/administration/emailConfiguration/EmailConfigurationPage.js:14]: please, check file indent

File: ui-tests/cypress/support/pageObjects/administration/emailConfiguration/EmailConfigurationPage.js [hunk 3]
```diff
@@ -1,3 +1,98 @@
...
+    return this.getContents()
+                .find(this.toolButton);
+}
+
+submit(){
+    return this.getToolButton()
+                .find(`${htmlElements.button}[type=submit]`);
+}
+
+//SenderManagement functions
```
Comment (g-di-martino) [inline ui-tests/cypress/support/pageObjects/administration/emailConfiguration/EmailConfigurationPage.js:93]: remove commented code; you can add it later once the sender management functions will be added... or you might end-up creating two separate files for the different sub-sections of the page...

File: ui-tests/cypress/support/restAPI/emailConfigAPI.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  });
+});
+
+class EmailConfigController{
+    
+    constructor(access_token) {
+        this.access_token = access_token;
+      }
+
+      options = {
```
Comment (g-di-martino) [inline ui-tests/cypress/support/restAPI/emailConfigAPI.js:17]: is this a constant? which options does it describe?

File: ui-tests/cypress/integration/administration/email-configuration.spec.js [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    
+    afterEach(() => {
+     
+        cy.emailConfigController().then(controller => controller.defaultSettings());
+        cy.kcLogout();
+
+      });
+
+
+    it('SMTP Current Configuration is displayed', () => {
```
Comment (g-di-martino) [inline ui-tests/cypress/integration/administration/email-configuration.spec.js:25]: rebase on `develop` and add the specific tags

**Review by g-di-martino — CHANGES_REQUESTED** [review]

File: ui-tests/cypress/support/restAPI/emailConfigAPI.js (hunk 4 above)
Comment (GianMartinoEntando) [reply ui-tests/cypress/support/restAPI/emailConfigAPI.js:17]: yes, it's a constant.  "smtpDefaultSettings" I think is better.

**Review by g-di-martino — APPROVED** [review]

**Pull request merged**